app/MMB/src/components/MissingInformation.js

```diff
@@ -16,14 +16,26 @@ import {
 import { withNavigation } from 'react-navigation';
 
 import Alert from '../components/alert/Alert';
+import BookingDetailContext from '../context/BookingDetailContext';
 import type { MissingInformation as PassengerType } from './__generated__/MissingInformation.graphql';
 
-type Props = {|
-  +data: PassengerType,
-  +navigation: NavigationType,
+type PropsWithContext = {|
+  ...Props,
+  setIsMissingDocumentId: (isMissingDocumentId: boolean) => void,
 |};
 
-export class MissingInformation extends React.Component<Props> {
+export class MissingInformation extends React.Component<PropsWithContext> {
+  componentDidMount = () => {
+    this.props.setIsMissingDocumentId(this.isPassengerMissingId());
+  };
+
+  isPassengerMissingId = () => {
```

Comment:
Maybe, is**Some**PassengerMissingId since it's related to passengers, and not just a single passenger.

Reply:
🚀